tests/robot-tests/tests/libs/utilities.py

```diff
@@ -251,10 +251,13 @@ def user_clicks_indicator_checkbox(indicator_label):
         f'//*[@id="filtersForm-indicators"]//label[contains(text(),"{indicator_label}")]/../input').click()
 
 
-def user_clicks_subheaded_indicator_checkbox(subheading_label, indicator_label):
-    sl.page_should_contain_checkbox(f'//*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input')
-    sl.driver.find_element_by_xpath(
-        f'//*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input').click()
+def user_selects_subheaded_indicator_checkbox(subheading_label, indicator_label):
```

Comment:
Couldn't all of these just be in `common.robot`?

The only complicated part is the selector, and even that can be assigned to a variable in Robot Framework.

Also, as this isn't being wrapped with the try/except, it wouldn't be great from a debug standpoint.